resources/crds/ripsaw_v1alpha1_hammerdb_index_cr.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha
+kind: Benchmark
+metadata:
+  name: hammerdb-benchmark
+  namespace: my-ripsaw
+spec:
+  # elasticsearch:
```

Comment:
add in clustername as well to the commented vars

Reply:
fixed